src/Fixer/Phpdoc/PhpdocOrderFixer.php

```diff
@@ -48,10 +48,35 @@ final class PhpdocOrderFixer extends AbstractFixer implements ConfigurableFixerI
 
     /**
      * @const string[]
-     *
-     * @TODO: 4.0 - change default to ['param', 'return', 'throws']
```

Comment:
If changing an order of 3 explicitly listed tags was considered as BC break and planned for v4, such drastic change in a default configuration of `phpdoc_order` is even more invasive for end users. It's fine to change this fixer's config in `PhpCsFixer` ruleset, but doing it directly in the fixer is most probably a no-go and need @keradus' decision.

Reply:
The original order is unchanged, more tags are added to be placed in a controlled order.

I belive this is wanted and allowed in new minor version by semver.

Reply:
ppl will see this as BC. we were there few times.

Reply:
> The original order is unchanged, more tags are added to be placed in a controlled order.

@mvorisek I know, I see the `@TODO` was moved to the line with tag that needs to be moved in the future, but the fact is this change would most probably enforce applying fixes in many repos and/or cause CI failures, as more tags are listed here (and now these are omitted?). I understand it's not a big deal in general and _for me_ it wouldn't be a problem at all, but @keradus has better knowledge about keeping BC historically and this concern was raised several times even for really small changes.

Reply:
I updated the PR to keep the default order but change only the `PhpCsFixer` config with TODO note it should be the default in v4. All words are satistified.